packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanTreeOffsetDuck.test.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { createStore } from 'redux';
+
+import reducer, { actions, newInitialState } from './SpanTreeOffsetDuck';
+
+describe('SpanTreeOffsetDuck', () => {
+  const existingSpanId = 'existingSpanId';
+  const newSpanId = 'newSpanId';
+
+  let store;
+
+  beforeEach(() => {
+    store = createStore(reducer, newInitialState());
```

Comment:
I would think you can test the actions without a store, or is uising a store  preference?